packages/client/src/pages/admin/integrations.vue

```diff
@@ -4,69 +4,52 @@
 		<FormFolder class="_formBlock">
 			<template #icon><i class="fab fa-twitter"></i></template>
 			<template #label>Twitter</template>
-			<template #suffix>{{ enableTwitterIntegration ? $ts.enabled : $ts.disabled }}</template>
+			<template #suffix>{{ enableTwitterIntegration ? i18n.ts.enabled : i18n.ts.disabled }}</template>
 			<XTwitter/>
 		</FormFolder>
 		<FormFolder to="/admin/integrations/github" class="_formBlock">
```

Comment:
I assume the `to` attribute was removed at some point, but the integrations were on their own pages previously.

Since you removed the page information in the components, it probably makes sense to remove these too:
```suggestion
		<FormFolder class="_formBlock">
```